<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>862</width>
    <height>623</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="control">
       <attribute name="title">
        <string>控制</string>
       </attribute>
       <widget class="QSlider" name="verticalSlider">
        <property name="geometry">
         <rect>
          <x>650</x>
          <y>130</y>
          <width>31</width>
          <height>301</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_connect">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>20</y>
          <width>71</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string>connect</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QLabel" name="label_speed">
        <property name="geometry">
         <rect>
          <x>640</x>
          <y>100</y>
          <width>53</width>
          <height>15</height>
         </rect>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_enable">
        <property name="geometry">
         <rect>
          <x>270</x>
          <y>20</y>
          <width>81</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string>Enable</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_right">
        <property name="geometry">
         <rect>
          <x>440</x>
          <y>210</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>right</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_front">
        <property name="geometry">
         <rect>
          <x>340</x>
          <y>160</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>front</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_back">
        <property name="geometry">
         <rect>
          <x>340</x>
          <y>270</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>back</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_left">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>210</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>left</string>
        </property>
       </widget>
       <widget class="QTableWidget" name="tableWidget_monitor">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>300</y>
          <width>401</width>
          <height>192</height>
         </rect>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <property name="columnCount">
         <number>4</number>
        </property>
        <column>
         <property name="text">
          <string>功能名称</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>类型</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>地址</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>值</string>
         </property>
        </column>
       </widget>
       <widget class="QLineEdit" name="lineEdit_IP">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>20</y>
          <width>113</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_port">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>50</y>
          <width>113</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>53</width>
          <height>15</height>
         </rect>
        </property>
        <property name="text">
         <string>IP:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>50</y>
          <width>53</width>
          <height>15</height>
         </rect>
        </property>
        <property name="text">
         <string>Port:</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>历史记录</string>
       </attribute>
       <widget class="QTableView" name="tableView_logs">
        <property name="geometry">
         <rect>
          <x>5</x>
          <y>11</y>
          <width>781</width>
          <height>501</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="detect">
       <attribute name="title">
        <string>检测</string>
       </attribute>
       <widget class="QGraphicsView" name="graphicsView">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>0</y>
          <width>611</width>
          <height>441</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_detectImg">
        <property name="geometry">
         <rect>
          <x>650</x>
          <y>210</y>
          <width>91</width>
          <height>91</height>
         </rect>
        </property>
        <property name="text">
         <string>detect</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="checkBox_show_ai">
        <property name="geometry">
         <rect>
          <x>660</x>
          <y>40</y>
          <width>91</width>
          <height>19</height>
         </rect>
        </property>
        <property name="text">
         <string>显示识别结果</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QCheckBox" name="checkBox_auto_read">
        <property name="geometry">
         <rect>
          <x>660</x>
          <y>70</y>
          <width>91</width>
          <height>19</height>
         </rect>
        </property>
        <property name="text">
         <string>自动读取</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_nextImg">
        <property name="geometry">
         <rect>
          <x>650</x>
          <y>350</y>
          <width>91</width>
          <height>81</height>
         </rect>
        </property>
        <property name="text">
         <string>next img</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="checkBox_showAiRect">
        <property name="geometry">
         <rect>
          <x>660</x>
          <y>100</y>
          <width>81</width>
          <height>19</height>
         </rect>
        </property>
        <property name="text">
         <string>锚框</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_cameraUrl">
        <property name="geometry">
         <rect>
          <x>650</x>
          <y>180</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>630</x>
          <y>180</y>
          <width>54</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>url</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>862</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
